winit-core/build.rs

```diff
@@ -0,0 +1,26 @@
+use cfg_aliases::cfg_aliases;
+
+fn main() {
```

Comment:
Nit: Might make sense to avoid this in `winit-core`, since it's intended to be platform-agnostic? But fine with it for now.

Reply:
I just put it there to not warn a lot, I still have to go through a lot of things like that and check.